src/test/resources/data/typicalDataBundle.json

```diff
@@ -817,6 +817,26 @@
       "isClosingEmailEnabled": true,
       "isPublishedEmailEnabled": true
     },
+    "publishedSession": {
```

Comment:
I think it is better to create the session programmatically in `GetSessionResultsActionTest ` rather than adding here as adding things in this file will cause a lot of changes which might hides bugs in the future.

Unless we can verify that all the changes caused by adding this session is intended.